pom.xml

```diff
@@ -128,8 +128,8 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-      <maven.compiler.source>1.7</maven.compiler.source>
-      <maven.compiler.target>1.7</maven.compiler.target>
+      <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
I am fine with either, I just pushed the PR against master but if we want to do 1.6 we can wait to merge it until after 1.5 is released.